tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -53,16 +53,36 @@ limitations under the License.
 #define gpuSuccess cudaSuccess
 using gpuStream_t = cudaStream_t;
 using gpuError_t = cudaError_t;
-
 #elif TENSORFLOW_USE_ROCM
 #define gpuSuccess hipSuccess
 using gpuStream_t = hipStream_t;
 using gpuError_t = hipError_t;
 #endif
 
-#define GetGPUStream(context) context->eigen_gpu_device().stream()
-
 namespace tensorflow {
+#if GOOGLE_CUDA
+// cudaGetErrorString is available to both host and device
+__host__ __device__ inline const char* gpuGetErrorString(cudaError_t error){
```

Comment:
This should be called 'GpuGetErrorString'. Maybe we can fix this in a follow-up PR?

Reply:
Oops another coding convention issue. Thanks for pointing out. This functions is rather isolated and has been applied to anywhere else. I'd rather file another commit that fix it now.

Looking at the CI result, I will add back the `GetCudaStream` function. I will file a follow up PR for making dependency changes from `GetCudaStream` to `GetGpuStream`.